src/PAModel/CanvasDocument.cs

```diff
@@ -422,8 +426,85 @@ internal HashSet<string> GetImportedComponents()
             return set;
         }
 
+
+
+        private void StabilizeAssetFilePaths()
+        {
+            _entropy.LocalResourceFileNames.Clear();
+
+            // Update AssetFile paths
+            foreach (var resource in _resourcesJson.Resources)
+            {
+                if (resource.ResourceKind != "LocalFile")
+                    continue;
+
+                var assetFilePath = FilePath.FromMsAppPath(resource.Path.Substring(AssetFilePathPrefix.Length));
```

Comment:
nit: can we wrap this part in a  function? We repeat it 4x, the substring on file path is brittle. 
```
FilePath.FromMsAppPath(resource.Path.Substring(AssetFilePathPrefix.Length));
```